File: 5.go
package main

import "fmt"

func main() {

	peserta := [4]int{18, 15, 10, 10}

	total := 0
	for i := 0; i < 4; i++ {
		total = total + peserta[i]
	}

	rataRata := float64(total) / float64(len(peserta))

	fmt.Println("Total seluruh peserta:", total)
	fmt.Println("Rata-rata peserta per Study Group:", rataRata)
}

File: 5.js
function main() {
    const peserta = [18, 15, 10, 10];

    let total = 0;
    for (let i = 0; i < 4; i++) {
        total = total + peserta[i];
    }

    const rataRata = total / peserta.length;

    console.log("Total seluruh peserta:", total);
    console.log("Rata-rata peserta per Study Group:", rataRata);
}

main();